src/views/PicTours/index.jsx

```
@@ -0,0 +1,191 @@
+            No Top Pic Tour Found
+          </div>
+        ) : null}
+      </div>
+
+      {/* Sub Categories and their Pic */}
+      <div className="mt-10">
+        {isPicTourFetching ? (
+          <Spinner />
+        ) : picTours?.length > 0 ? (
+          picTours?.map((pictour) => (
+            <div className="mb-5">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)